packages/playwright/src/common/fixtures.ts

```diff
@@ -193,12 +200,29 @@ export class FixturePool {
 
     const hash = crypto.createHash('sha1');
     const names = Array.from(this._registrations.keys()).sort();
+
+    // First iterate over non-boxed fixtures to provide clear error messages.
     for (const name of names) {
       const registration = this._registrations.get(name)!;
-      visit(registration);
+      if (registration.box)
+        continue;
+      visit(registration, true);
```

Comment:
I find it confusing that we pass `box=true` when `registration.box === false`. Perhaps rename the argument?

Reply:
It is `boxed`, not `box`. I'll rename to `onlyBoxed`